### PR TITLE
transfer cufflinks from devteam and fix

### PR DESCRIPTION
**Pull request by bernt-matthias**
started here: https://github.com/bernt-matthias/tools-iuc/pull/new/topic/cufflinks

@nsoranzo suggested to transfer to IUC

- [x] could someone check the `shed.yml` files

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by nsoranzo** [comment]
Can you rebase on top of the master branch? So cummerbund.R gets checked before merge.

**Comment by bernt-matthias** [comment]
1 linter error per line. happy that it was only 2 lines of R code :)

File: tool_collections/cufflinks/cuffnorm/cuffnorm_wrapper.xml [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+            ## Set advanced parameters for cufflinks
+            #if $advanced_settings.sAdvanced == "Yes":
+                #if str($advanced_settings.library_type) != 'auto':
+                        --library-type=$advanced_settings.library_type
+                #end if
+                $advanced_settings.hits_norm
+            #end if
+            ## Inputs.
+            $gtf_input
```
Comment (nsoranzo) [inline tool_collections/cufflinks/cuffnorm/cuffnorm_wrapper.xml:25]:
```suggestion
            '$gtf_input'
```

**Comment by nsoranzo** [comment]
Ops, I forgot about the permissions!

@martenson @davebx Can you please give iuc the usual permissions on TTS and TS for the following 6 devteam's repositories: cuffcompare, cuffdiff, cufflinks, cuffmerge, cuffnorm, cuffquant

**Comment by martenson** [comment]
granted, thanks for the ping :+1:

**Comment by nsoranzo** [comment]
> granted, thanks for the ping +1

Thank you @martenson ! Uploads were successful this time.

**Pull request merged**